### PR TITLE
enforce_no_element_style

### PR DESCRIPTION
**Pull request by ulfgebhardt**
## 🍰 Pullrequest

Enforce style not to be defined on the element but external at least in a `<style>` tag

### Issues

- None

### Todo

- [X] None

### CODE REVIEW

File: admin/src/pages/UserSearch.vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="user-search">
-    <div style="text-align: right">
+    <div id="user-search-style-1">
```
Comment (Mogge) [inline admin/src/pages/UserSearch.vue:3]: serious?

File: frontend/src/views/Pages/AccountOverview/GddTransactionList.vue [hunk 2]
```diff
@@ -212,4 +208,18 @@ export default {
 .gdd-transaction-list-item {
   outline: none !important;
 }
+
+.red-icon {
+  color: red;
+}
+
+#collaps-button-style {
```
Comment (Mogge) [inline frontend/src/views/Pages/AccountOverview/GddTransactionList.vue:216]:
```suggestion
.collaps-button-style {
```

File: frontend/src/views/Pages/AccountOverview/GddTransactionList.vue [hunk 3]
```diff
@@ -212,4 +208,18 @@ export default {
...
+  width: 95%;
+  position: absolute;
+}
+
+#collapse-div-style-1 {
```
Comment (Mogge) [inline frontend/src/views/Pages/AccountOverview/GddTransactionList.vue:221]:
```suggestion
.collapse-div-style-1 {
```

**Review by Mogge — CHANGES_REQUESTED** [review]
Very quick and dirty :(

**Comment by ulfgebhardt** [comment]
> Very quick and dirty :(

I think its adequate - it was you who said we do not invest into style, since we get a new design in the near future anyways. It is expected that all of those style- and the html tags will change completely. I believe its important to have this check in place before we do the design implementation.

**Pull request closed without merging**